modules/cf-types/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 80,
+  "trailingComma": "none"
+}
```

Comment:
Seems unrelated to this PR.

Reply:
it was to fix the linting from when we added these packages into our repo, but yes otherwise unrelated to this PR

Reply:
Correct. This has been cherry-picked into other branches as well. Will leave it for simplicity.